_posts/2019-02-14-customize.md

```diff
@@ -907,125 +909,112 @@ $ yq w -i $TARGET public.app.clientTitle "New Title"
+| `userdata-bbb_auto_share_webcam=`        | If set to `true`, the client will start the process of sharing webcam (if any) automatically upon loading the client | `false`       |
+| `userdata-bbb_preferred_camera_profile=` | Specifies a preferred camera profile to use out of those defined in the `settings.yml`                               | none          |
+| `userdata-bbb_enable_screen_sharing=`    | If set to `false`, the client will display the screen sharing button if they are the current presenter               | `true`        |
+| `userdata-bbb_enable_video=`             | If set to `false`, the client will display the webcam sharing button (in effect disabling/enabling webcams)          | `true`        |
+| `userdata-bbb_enable_video_stats=`       | If set to `true`, the client will display connection statistics for the user                                         | `false`       |
+| `userdata-bbb_skip_video_preview=`       | If set to `true`, the client will not see a preview of their webcam before sharing it                                | `false`       |
 
-* `userdata-bbb_auto_swap_layout=true`
-(default value is `false`)
 
-TODO
+### Presentation parameters
 
-* `userdata-bbb_hide_presentation=true`
-(default value is `false`)
+| Parameter                                                | Description                                                                                                                                                                      | Default value |
+| -------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `userdata-bbb_force_restore_presentation_on_new_events=` | If set to `true`, new events related to the presentation will be pushed to viewers. See [this PR](https://github.com/bigbluebutton/bigbluebutton/pull/9517) for more information | `false`       |
 
-If set to `true`, the presentation area will be minimized automatically
+### Whiteboard parameters
 
-* `userdata-bbb_listen_only_mode=false`
-(default value is `false`)
+| Parameter                           | Description                                                                                                    | Default value |
+| ----------------------------------- | -------------------------------------------------------------------------------------------------------------- | ------------- |
+| `userdata-bbb_multi_user_pen_only=` | If set to `true`, the whiteboard will be multi-user only, meaning that all participants can annotate it freely | `false`       |
+| `userdata-bbb_presenter_tools=`     | If set to `false`, the client will display the presenter tools                                                 | `true`        |
```

Comment:
You can pass an array with the tools that you want to be available to the presenter.